src/foam/nanos/menu/VerticalMenu.js

```diff
@@ -80,14 +80,14 @@ foam.CLASS({
       }
     },
     {
-      name: 'menuSearch',
       class: 'String',
-      value: '',
+      name: 'menuSearch',
       view: {
-        class: 'foam.u2.view.TextField',
-        onKey: true,
-        type: 'Search'
-      }
+        class: 'foam.u2.TextField',
```

Comment:
Fixed autocomplete for Safari, but problem remains for Chrome
even if specify that email field during sign in should be of type 'email'.

I'm not sure if it can be fixed
https://support.google.com/chrome/thread/6787468?hl=en

Reply:
Does adding this work?
  autocomplete="off"